src/site/_includes/components/EventTable.js

```diff
@@ -41,17 +42,48 @@ module.exports = (event, authorsCollection) => {
     }
   }
 
-  const renderSession = ({speaker, title}) => {
+  const slugs = {};
+  const slugForTitle = (title) => {
+    // Find a slug for this title, but prevent duplicate IDs.
+    const base = slugify(title, {
+      lower: true,
+      strict: true,
+      remove: /[^-\w _]/, // remove anything not in: basic word chars, space, - and _
+    });
+    let id = base;
+    let suffix = 0;
+    while (id in slugs) {
+      id = base + ++suffix;
+    }
+    slugs[id] = title;
+    return id;
+  };
+
+  const renderSession = ({speaker, title, abstract}) => {
     // Always pass an Array of author IDs.
     const authors = typeof speaker === 'string' ? [speaker] : speaker;
 
+    const id = slugForTitle(title);
+
+    // Coerce to array or empty array.
+    abstract =
+      (abstract && (typeof abstract === 'string' ? [abstract] : abstract)) ||
+      [];
+
     return html`
-      <tr>
-        <td class="w-event-schedule__speaker">
+      <div class="w-event-schedule__row" data-session-id=${id}>
+        <div class="w-event-schedule__cell w-event-schedule__speaker">
           ${AuthorsDate({authors}, authorsCollection)}
-        </td>
-        <td class="w-event-schedule__session">${title}</td>
-      </tr>
+        </div>
+        <div class="w-event-schedule__cell w-event-schedule__session">
+          <a class="w-event-schedule__open" href="#${id}">
+            <span>${title}</span>
+          </a>
+          <div class="w-event-schedule__abstract" hidden>
+            ${abstract.map((part) => html`<p>${part}</p>`)}
```

Comment:
I was curious if we should pull in markdown-it and run the abstract text through. That way folks could include links (Phil and Paul Lewis's abstract contains a link).

Reply:
On second thought, if folks want to add a link we can just have them include the html in their abstract. I'll update phil and paul's talk to do so.